frontend/www/js/omegaup/components/problem/StatementEdit.vue

```diff
@@ -63,9 +63,9 @@
                     >{{ T.wordsProblemsetter }}:
                     <a class="problemsetter">
                       <omegaup-user-username
-                        v-bind:classname="classname"
+                        v-bind:classname="problemsetter.classname"
```

Comment:
```suggestion
                        v-if="problemsetter"
                        v-bind:classname="problemsetter.classname"
```

si no, puede haber un acceso a `null` inválido.

Reply:
Listo.